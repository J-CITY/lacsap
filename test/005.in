__a daw_q
