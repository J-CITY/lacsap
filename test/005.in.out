__a   0   0      identifier    24
daw_q   0   4      identifier    24
